netbeans.apache.org/src/content/kb/docs/java/code-inspect.adoc

```diff
@@ -43,56 +43,19 @@ In this tutorial, you will learn how to perform static analysis of your Java cod
 
 |link:https://web.archive.org/web/20131021173103/https://netbeans.org/project_downloads/samples/Samples/Java/library.zip[library.zip] |NetBeans project 
 
-|link:http://repo1.maven.org/maven2/com/google/code/findbugs/jsr305/2.0.0/jsr305-2.0.0.jar[jsr305-2.0.0.jar] |Java Specification Request intended to improve the effectiveness of static analysis tools 
-|===
```

Comment:
@ebarboni @vieiro this might be the problem right? The removal of `|===` breaks the table?